modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -1,68 +1,177 @@
+
+For details, see xref:defining-sla-tiers.adoc[Reviewing SLA Tiers Concepts].
+
+=== Client Providers (Identity Providers)
 
-// talk about naming, instances
+You use client providers to enforce security and regulations in your business organization. Client providers authorize client applications. 
 
-After configuring the API, you can perform API management tasks, applying policies and setting up SLA tiers, assigning permissions to the API environment,
-versioning, viewing API analytics, and more.
+For details, see xref:configure-multiple-credential-providers.adoc[Configure Multiple Client Providers for Client Management].
 
-The API dashboard shows key information about a deployed API instance:
+=== Contracts
+
+Contracts define how client applications consume APIs. A client application requests access in Exchange. Either the owner of the API instance approves the request in API Manager, or the approval is automatic, depending on configuration. 
+
+Contracts are enforced with either of the following:
+
+* SLA enforcement policies
+* Client enforcement policies
+
+For details, see xref:api-contracts-landing-page.adoc[Client Applications and Contracts].
+
+=== API Alerts
+
+An API alert (different from a Runtime Manager alert) is an alarm that flags one of the following:
+
+* The API request violates a policy.
+* Requests received by the API exceed a given number within a specified period of time.
+* The API returns a specified HTTP error code.
+* The API response time exceeds a specified timeout value.
+
+API Manager triggers alerts when states change from desirable to undesirable or vice versa. When an alert is triggered, API Manager sends an email notification to you and to anyone else you specify in the configuration.
+
+For details, see xref:using-api-alerts.adoc[Alerts].
+
+=== API Assets
+
+API assets, published by Exchange, are the components that comprise applications. Components include any of the following:
+
+* APIs (OAS, RAML, RAML fragments, HTTP, WSDL, and custom)
+* API groups
+* Policies
+* Examples
+* Templates
+* Modules
+* Connectors
+
+For details, see xref:exchange::about-sharing-assets.adoc[Share Assets].
+
+=== API Groups
+
+An API group is a collection of API instances that act as a single unit, so that applications can access the APIs as a group using one client ID. API groups are created in API Manager and published to Exchange.
+
+For details, see xref:api-groups-landing-page.adoc[API Groups].
+
+=== API Instances
+
+An API instance belongs to one environment, but you can have different API instances of an API in different environments.
```

Comment:
I can't seem to wrap my head around this: do we mean...

API B = Environment X, but
API Ba = Environment Y
API Bb = Environment Z....

Or are we saying that any one API instance can belong to only one environment, but that one environment can "own" more than one API instance?

One API : API A
Multiple instances of API A : API A1, API A2, API A3..., each belonging to only one environment
One environment can own multiple API instances, but those instances belong only to that one environment?

Reply:
(If this is something that your audience will readily understand, then pass over my question...but do you really want to say:
```suggestion
One API instance belongs to one environment, but you can have different instances of that same API in different environments.
```

Reply:
I decided it was best to remove this sentence, as I'm thinking it distracts from the intent of the topic (which is to describe the UI). The sentence feels misplaced. Furthermore, the information in the sentence is included in the topic's link below the screenshot.